### ISSUE TITLE
Acces to Data - suggestions

### ISSUE THREAD

**Issue by MpaulaL** [esciencecenter-digital-skills/Lesson-FAIR-Data-Climate#53]
- I think that objectives and questions should be revised here
- I don't see information on 'How to choose a data repository?' and maybe it is too early for that without talking about PIDs and licences? I do see the exploratory exercise of repositories, which I like, but the questions should be really focused on accessibility 
- I don't see DMPs mentioned despite this objective: Plan access to the data in a data management plan. I would actually skip this is DMPs have not been introduced. And, all the FAIR principles shuold be planned in a DMP
-  For consistency I would add in all section the "what does go-FAIR says"or remove it form here
- Change "FAIRness in climate science" for Access to data in Climate science, it is confusing to talk about FAIRness as a whole when we are focusing on access
- I miss more explicit explanation between data access and data publication/repositories
- Accessible vs open - Accessible is defined, but open is not. That might clarify the difference
- What data should not be open, but metadata can be accessible?
- The exercise on stages of accessibility I would rephrase it  - organize the options from the least to the most FAIR way to make data accessible, reflect and discuss why you chose that particular order
- A challenging database to looking at accessibility and re-usability permissions is this: https://www.ecmwf.int/en/forecasts/accessing-forecasts It could be used as an example or it could added a a repo to explore

**Comment by SarahAlidoost** [comment]
@MpaulaL thank you for your valuable suggestions. I couldn't understand your point in the item _"What data should not be open, but metadata can be accessible?"_  Could you please elaborate on this?

**Comment by MpaulaL** [comment]
Thanks @SarahAlidoost  for incorporating my suggestions! with that comment I mean that maybe it would be good to give examples about data that shouldn't be open, are there examples in climate data we could give? and if the data can't be made open, is it still good practice to publish a metadata record with the contact information of the responsible person for the data. Data that can't be share openly, should still be available upon request, but for re-users to know that the data exist, there should be a record of it and a metadata record helps with that and makes the data a bit FAIRer. 
Please let me know if the explanation makes sense?

**Comment by SarahAlidoost** [comment]
> Thanks @SarahAlidoost for incorporating my suggestions! with that comment I mean that maybe it would be good to give examples about data that shouldn't be open, are there examples in climate data we could give? and if the data can't be made open, is it still good practice to publish a metadata record with the contact information of the responsible person for the data. Data that can't be share openly, should still be available upon request, but for re-users to know that the data exist, there should be a record of it and a metadata record helps with that and makes the data a bit FAIRer.
> Please let me know if the explanation makes sense?

Thank you for your explanations. I addressed your comments in this pull request (#54) that is ready for your review.
